recipes/matplotlib-cpp/all/conandata.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  cci.20200422:
+    - url: https://raw.githubusercontent.com/lava/matplotlib-cpp/d74105036a82e6ccede3fa1a7d872ac1677b00ab/matplotlibcpp.h
+      sha256: 913638b1d03ccfe28f58f327a8c1247aec943f88f19c37648fd01e239420fb90
+    - url: https://raw.githubusercontent.com/lava/matplotlib-cpp/d74105036a82e6ccede3fa1a7d872ac1677b00ab/LICENSE
+      sha256: d0dc4eae3be039d576e127912d16f78a7ba4e0e6205cd75103efd1a45b89a4ba
```

Comment:
Is it not less maintenance effort with https://github.com/lava/matplotlib-cpp/archive/d74105036a82e6ccede3fa1a7d872ac1677b00ab.zip?

Reply:
Why this "old" commit by the way?

Reply:
Uh, how did you get that, did not see a release :-/

Reply:
github provides a way to download a zip of any commit.

Reply:
> Why this "old" commit by the way?

I `git bisected` this version to be the last one that supports C++11, followups need C++17

Reply:
done (the download part)